src/content/docs/project/runs/managing-runs.md

```diff
@@ -178,17 +181,95 @@ Here are steps: 1. Open Run Group 2. Click Extra menu 3. Click Copy 4. Pick conf
 
 ![Copy group 3](./images/34.png)
 
-## Merge Strategy
+## Merge Strategies
+
+Testomat.io provides different **'Merge Strategies'** to aggregate the results of the individual test runs within a Run Group. This is crucial when you have multiple executions of the same tests (e.g., retries, parallel execution, runs on different environments) and want a single, representative status for the tests within the group.
+
```

Comment:
For instance, let's say we have 2 runs inside a rungroup with the same tests A and B:

* Run 1: A - failed, B - failed
* Run 2: A - passed, B - passed

What counter numbers should contain Rungroup? Both answers: "Passed: 2, Failed: 0" or "Failed 2, Passed 0" are absolutely valid depending on how you calculate those numbers. That's why Testomat.io provides customizable Merge strategies:

* Optimistic - prefer passed over failed (`Passed: 2, Failed: 0` in our example)
* Pessimistic - prefer failed over passed (`Passed: 0, Failed: 2` in our example)
* Realistic - prefer the result from the last run (`Passed: 2, Failed: 0`, as Run 2 was added later in our example)

Reply:
Add a brief sample description before explaining merge strategies ^